anago

```diff
@@ -203,25 +203,29 @@ check_prerequisites () {
        (logrun gcloud config set account $user && \
         logrun gcloud docker -- version >/dev/null 2>&1); then
       logecho -r "$OK"
+      break
```

Comment:
I think we might be checking both because mock runs push to something ACL'd to the user running anago, whereas real runs push to a location only $G_AUTH_USER can access (https://github.com/kubernetes/release/blob/master/lib/releaselib.sh#L793).

I agree that we can probably do something smarter than what we're doing right now.